web/src/stream_settings_data.ts

```diff
@@ -37,6 +44,7 @@ export function get_sub_for_settings(sub: StreamSubscription): SettingsSubscript
             "dayofyear_year",
         ),
         creator: stream_data.maybe_get_creator_details(sub.creator_id),
```

Comment:
since we are already calling `stream_data.maybe_get_creator_details(sub.creator_id)` at line 34, we may not need to call it again.